modules/heart/heart.js

```diff
@@ -0,0 +1,64 @@
+import Component from "@pencil.js/component";
+
+/**
+ * Heart class
+ * @class
+ * @extends Component
+ */
+export default class Heart extends Component {
+    /**
+     * Heart constructor
+     * @param {PositionDefinition} positionDefinition - Position of the center of the heart
+     * @param {Number} radius - radius The distance from center of heart
+     * @param {ComponentOptions} [options] - Drawing options
+     */
+    constructor (positionDefinition, radius, options) {
+        super(positionDefinition, options);
+
+        if (radius > 0) {
+            this.radius = radius;
+        }
+        else {
+            this.radius = 1;
+        }
+    }
+
+    /**
+     * @inheritDoc
+     */
+    toJSON () {
+        const json = super.toJSON();
+        delete json.radius;
+        return json;
+    }
+
+    /**
+     * Draw the arc
+     * @param {Path2D} path - Drawing context
+     * @return {Heart} Itself
+     */
+    trace (path) {
+        path.moveTo(75 * this.radius, 40 * this.radius);
+        path.bezierCurveTo(75 * this.radius, 37 * this.radius, 70 * this.radius, 25 * this.radius,
+            50 * this.radius, 25 * this.radius);
+        path.bezierCurveTo(20 * this.radius, 25 * this.radius, 20 * this.radius, 62.5 * this.radius,
+            20 * this.radius, 62.5 * this.radius);
```

Comment:
First possibility, you can pu each params on its own line.
```suggestion
    path.bezierCurveTo(
        20 * this.radius,
        25 * this.radius, 
        20 * this.radius,
        62.5 * this.radius,
        20 * this.radius,
        62.5 * this.radius,
    );
```

Second one, you can create variable for your params.
```suggestion
    const to = [20 * this.radius, 25 * this.radius];
    const ctrl1 = [20 * this.radius, 62.5 * this.radius];
    const ctrl2 = [20 * this.radius, 62.5 * this.radius];
    path.bezierCurveTo(...to, ...ctrl1, ...ctrl2);
```

The second solution is good because you seems to reuse a lot of values.

Reply:
I think I might just go with the first option. There's not that many repeating variables and I rather not flip flop around.